frontend/src/Requirements.css

```diff
@@ -82,4 +82,8 @@
 
 .fa-search:hover {
   color: #e74c3c;
-}
\ No newline at end of file
+}
+
+.fa-exclamation-circle{
+  color: #c13424;
```

Comment:
We're using a standard red already: #e74c3c

Reply:
Changed